clients/privacy-center/public/fides-js-demo.html

```diff
@@ -4,7 +4,7 @@
     <title>fides-js script demo page</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <script src="/fides.js"></script>
+    <script src="/fides-js"></script>
```

Comment:
this allows testing against the PC endpoint instead of just the bundle (which we need in order for prefetch to happen!)

Reply:
wait, could you explain this a bit more? I thought `fides.js` includes the prefetch code as part of the bundle. Since our customers use `fides.js` instead of calling the PC endpoint directly I want to ensure we're best mimicking their setup on our `fides-js-demo.html` page

Reply:
you know what, you're right... I could've sworn it wasn't working before I changed it to `fides-js`, but now that I think about it, that doesn't really make sense either. I'll revert!

Reply:
okay I fixed, and also added a treat where you can now add a query param to force a geolocation (similar to what we do for cookiehouse) https://github.com/ethyca/fides/pull/4124/commits/cc1c94d11e5bbe64e36aa2e5dbd6c7dfcba84b26